cli/src/commands/create-expo-stack.ts

```diff
@@ -127,6 +127,9 @@ const command: GluegunCommand = {
 						type: 'authentication'
 					});
 				}
+				if (options.vexo) {
```

Comment:
@falcoagustin can you add this to the --help option? should be here https://github.com/danstepanov/create-expo-stack/blob/main/cli/src/utilities/showHelp.ts